DiveVR/DiveVR
# [DVR2-9] Add DynamoDB to Project

Changes:
- Installed `uuid` package
- Add `createUserEntry()` and `addVideo() functions` for storing user info in DynamoDB

To Test:
1. Pull this branch
2. In the root directory, do `npm install`
3. Download the updated `.env` file from Google Drive
4. Make sure your `.aws` folder is deleted or else the credentials will conflict. See https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html for details
5. Run `npm start` and open http://localhost:3001/ in a browser

The program will automatically generate a random UUID and create an item in DynamoDB. It will also add 2 random videos to that user. You should get a JSON response like this:

![image](https://user-images.githubusercontent.com/36905285/69012362-dc25df00-0929-11ea-9957-e43891f592af.png)

Log into the AWS Console and verify the DynamoDB table has the same values added
